### DISCUSSION TITLE
On iOS, I’d like the Right Hand Tag List to be Sorted Alphabetically

### DISCUSSION THREAD

**Discussion by Pendyploo** [standardnotes/forum#3105] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Pendyploo** [comment]
On iOS, I’d like the right-hand Tag List to be sorted alphabetically.
If you want to assign additional tags to a note, and you bring up the RHS Tag List, you find that it's not sorted alphabetically, so finding the Tag you need is very difficult and time-consuming.
If the Tag you need is a Sub-Tag, you often can't see it on a phone or iPad, because the Tag List doesn't word-wrap, and the Sub-Tag is off the RHS of the list.

**Reply by cronkish** [reply]
Second this, greatly diminishes the utility of tags to not have them sorted alphabetically.
